### ISSUE TITLE
Sky NMs have incorrect behavior

### ISSUE THREAD

**Issue by dvalderrama89** [OmegaPrivateServer/issues#200]
## Summary

The NMs listed in Expected Behavior are not working as intended

## Expected Behavior

1. Despot - Should spawn claimed to the PH killer and at the location of the PH onDeath
2. Faust - Should "patrol" the 2 long hallways while stationary in the spawn room and have a long aggro range. Should also spam Typhoon.
3. Brigandish Blade - Should be unkillable by anything other than the knife which you steal off him
4. Zipacna - Should patrol the hallway connecting Byakko and Genbu islands in a set pattern based on which doors are open/closed
5. Steam Cleaner - Should spawn from one of the Detectors located around Ve'Lugannon Palace

## Current Behavior

Basically none of the behavior in Expected is working

## Relevant Wiki Link 
(IMPORTANT: AUGUST 2008 WIKI ENTRIES OR EARLIER)

https://ffxiclopedia.fandom.com/wiki/Despot?oldid=641079
https://ffxiclopedia.fandom.com/wiki/Faust?oldid=647262
https://ffxiclopedia.fandom.com/wiki/Brigandish_Blade?oldid=632329
https://ffxiclopedia.fandom.com/wiki/Zipacna?oldid=636456
https://ffxiclopedia.fandom.com/wiki/Steam_Cleaner?oldid=632407

**Comment by zierk** [comment]
A fix for despot has been merged into the next patch. I'll work on the rest of these NMs plus some additional fixes which are not listed here over time.

**Comment by zierk** [comment]
Faust has been updated.
